migrate-dao.js

```diff
@@ -508,6 +510,8 @@ async function migrateDAO ({ web3, spinner, confirm, opts, migrationParams, logT
           schemeParams.push(votingMachinesParams[customeScheme.params[i].voteParams])
         } else if (customeScheme.params[i] === 'GenesisProtocolAddress') {
           schemeParams.push(GenesisProtocol)
+        } else if (customeScheme.params[i] === 'AvatarAddress') {
```

Comment:
it is not already been pushed at https://github.com/daostack/migration/blob/b3019efb8d70d0e9f3a67ec06a8d21c40832832a/migrate-dao.js#L507 ?

we can say require that that each none universal scheme should pass the avatar as the first argument to the inititilize function.